hummingbird/ml/operator_converters/__init__.py

```diff
@@ -30,6 +30,8 @@
 from .sklearn import gbdt  # noqa: E402
 from .sklearn import iforest  # noqa: E402
 from .sklearn import linear as sklearn_linear  # noqa: E402
+from .sklearn import mlp as sklearn_mlp  # noqa: E402
+from .sklearn import nb as sklearn_nb  # noqa: E402
```

Comment:
But now that we are keeping the _mlp_implementations.py and _nb_implementations.py , we  need to add that to lines 48 of this file for pdoc to work. :)

Reply:
(other than that i think we are ready to merge!   :) )

Reply:
Ok. I will change and update the pull request.